src/components/Observations/GridItem.js

```diff
@@ -1,118 +1,114 @@
 
-  const totalObsPhotos = item?.observationPhotos?.length;
+  const totalObsPhotos = observation?.observationPhotos?.length;
   const hasMultiplePhotos = totalObsPhotos > 1;
-  const filterIconName = totalObsPhotos > 9 ? "filter-9-plus" : `filter-${totalObsPhotos}`;
+  const hasSound = !!observation?.observationSounds?.length;
+  const filterIconName = totalObsPhotos > 9 ? "filter-9-plus" : `filter-${totalObsPhotos || 2}`;
 
   const imageUri = uri === "project"
-    ? Observation.projectUri( item )
+    ? Observation.projectUri( observation )
     : { uri: Photo.displayLocalOrRemoteMediumPhoto( photo ) };
 
-  const showStats = ( ) => {
-    if ( uri !== "project" && item.needsSync( ) ) {
+  const showStats = () => {
+    if ( uri !== "project" && observation.needsSync() ) {
       return (
         <View className="absolute bottom-0 right-0">
-          <UploadButton observation={item} />
+          <UploadButton observation={observation} />
         </View>
       );
     }
-    const showUpload = uri !== "project" && item.needsSync( );
     return (
-      <View
-        className={classnames( "absolute bottom-0", {
-          "right-0": showUpload
-        } )}
-      >
-        {showUpload ? (
-          <UploadButton observation={item} />
-        ) : (
-          <ObsStatus
-            observation={item}
-            layout="horizontal"
-            color={colors.white}
-          />
-        )}
-      </View>
+      <ObsStatus observation={observation} layout="horizontal" color={colors.white} />
     );
   };
 
   return (
     <Pressable
       onPress={onPress}
-      className={`w-1/2 px-4 py-2 ${
-        ( index || 0 ) % ( numColumns || 2 ) === 0 ? "pr-2" : "pl-2"
-      }`}
-      testID={`ObsList.gridItem.${item.uuid}`}
+      className="h-[172px] w-[172px] rounded-[17px] overflow-hidden"
+      testID={`ObsList.gridItem.${observation.uuid}`}
       accessibilityRole="link"
       accessibilityLabel={t( "Navigate-to-observation-details" )}
     >
-      <View className="relative">
+      <View className="rounded-[17px] overflow-hidden relative">
         {imageUri && imageUri.uri ? (
-          <Image
+          <ImageBackground
             source={imageUri}
             className="grow aspect-square"
             testID="ObsList.photo"
-          />
+          >
+            <LinearGradient
+              className="bg-transparent absolute inset-0"
+              colors={["rgba(0, 0, 0, 0)", "rgba(0, 0, 0, 0.5) 100%)"]}
+            />
+          </ImageBackground>
         ) : (
-          <View className="grow aspect-square justify-center items-center">
-            <IconMaterial name="image-not-supported" size={150} />
-          </View>
+          <LinearGradient
+            colors={["rgba(0, 0, 0, 0)", "rgba(0, 0, 0, 0.5) 100%)"]}
+          >
+            <View className="grow aspect-square justify-center items-center">
+              <IconMaterial name="image-not-supported" size={150} />
```

Comment:
If the observation has no photos, don't show this `image-not-supported` icon. We just want that gray gradient.